command/agent/approle_end_to_end_test.go

```diff
@@ -400,6 +401,53 @@ func testAppRoleEndToEnd(t *testing.T, removeSecretIDFile bool, bindSecretID boo
 	}
 }
 
+// Prior to VAULT-8518 being fixed, you were unable to delete an approle value longer than 1024 bytes
```

Comment:
Thanks for adding a go test for this 🌟! Super minor nit here, but if we want this to exposed to godocs, don't we want to start the comment with `TestAppRoleLongRoleName`?